azclishell/az_completer.py

```diff
@@ -131,6 +131,7 @@ def get_completions(self, document, complete_event):
 
         text = reformat_cmd(text)
         if text.split():
+
```

Comment:
Haha, I don't know why it's doing that, but it does. In commit: https://github.com/Azure/azure-cli-shell/pull/40/commits/41c56d53893f7364c5baac332d6dbf8557d8908c this is the bulk of what I did.